config/routes.rb

```diff
@@ -2016,6 +2016,7 @@
         download_data
         download_summary_pdf
         edit
+        index
```

Comment:
I am little confused about introducing this new route here just for breadcrumbs, can you please explain why this is needed.

Reply:
Replaced with `show_list` with params.